packages/contracts/src/components/IdentityComponent.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0;
+
+import "std-contracts/components/StringComponent.sol";
+
+uint256 constant ID = uint256(keccak256("ds.component.Name"));
```

Comment:
`ds.` is what Dark Seas used to differentiate their world component names - let's just be sure to remove it before pushing